scripts/docker.build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -ex
+
+SOURCE_DIR=$(pwd)
```

Comment:
Use `"$( dirname -- "${BASH_SOURCE[0]}" )"` instead? Also, let's not depend on your home path :)